server/routerlicious/packages/services-shared/src/storage.ts

```diff
@@ -147,14 +148,32 @@ export class DocumentStorage implements IDocumentStorage {
 			lumberjackProperties,
 		);
 		let initialSummaryVersionId: string;
+		let initialStorageName: string | undefined;
```

Comment:
>initialStorageName [](http://example.com/codeflow?start=6&length=18)

(nit) just use `storageName`? `initialStorageName` gives me an impression that we have other non-initial storage names.